### DISCUSSION TITLE
cannot run  AUTO ORTHO

### DISCUSSION THREAD

**Discussion by limumeng** [kubilus1/autoortho#354] [General]
I downloaded AutoOrtho , added all sceneries. And selecting Run - 
Aftwer a series of  INFO: it ends with {'cur_mem_mb': 101}  message...and keep repeating. What is wrong ? Thanks anyone who can assist.

![Screenshot 2023-08-28 180940](https://github.com/kubilus1/autoortho/assets/76612779/7cb1cce5-55ad-4726-b3b3-63cbdab89a05)
![Screenshot 2023-08-28 182045](https://github.com/kubilus1/autoortho/assets/76612779/423828d1-1a35-416d-b13c-e53764c13ad5)

**Comment by NNowakowski** [comment]
Nothing's wrong. This is exactly what you want to see.
At this point just start XP and enjoy.

**Reply by limumeng** [reply]
Thanks, OK, ,  when I start X plane 12, how do i know Auto Ortho is running or the XPlane scenery is running?

**Comment by NNowakowski** [comment]
That should be pretty obvious. Just start a flight and you'll either see
ortho scenery or not.
Just keep the console window running in the background and let it do its
job.
Make sure to arrange the custom scenery ini accordingly or else you'll be
missing buildings.

**Reply by limumeng** [reply]
Ok. I followed your instructions. And has this error.
![IMG_3195](https://github.com/kubilus1/autoortho/assets/76612779/236bc34e-0598-4464-b862-ca23bf46ccd6)

**Reply by DemonTraitor** [reply]
yup, that's as far as I get

**Comment by marloscaida** [comment]
How-to instructions for installing AutoOrtho for Windows 10/11.

Many people cannot use AutoOrtho because they have not done a good installation or have not understood the instructions. This is one more attempt to give you a little help so that this wonderful application works properly for you and you can enjoy it.

You must make sure that your operating system is the 64-bit version.
Works in X-Plane 11 or 12

If you don't have a good machine, with a good video card and fast drives, don't even bother reading the instructions. AutoOrtho is definitely not for you. If you have, congratulations. Follow these steps. If you don't understand something or don't dare to do it, ask a friend who knows a little more about Windoes and X-Plane for help.

1.- Obtain any of the following products:

-Dokan
1.1. Download and install the latest version of Dokan here:
https://github.com/dokan-dev/dokany/tags
Use DokanSetup.exe or Dokan_x64.msi

-WinFSP
1.2. Download and install the latest version of WinFSP here:
https://github.com/winfsp/winfsp/tree/v2.0
(We recommend downloading the MSI file for installation)

2.- Install one of the two products in a standard or default way, without changing anything during the installation.

(If it doesn't work on your Dokan system, uninstall it and try WinFSP. You have to reboot your computer, every time you install or uninstall)

In my case, DOKAN didn't work for me, but it did for my friends.
I installed WinFSP and it worked (I have no idea why).
These libraries are used to create a virtual folder that only exists while AO is running.

3.- Get the latest version of AutoOrtho here:

https://github.com/kubilus1/autoortho/tags

Use version AutoOrtho_win_xxx.zip (manual installation)

4.- Uncompress the zipped file in the place you want (it can be on your main drive, it can be inside the root of X-Plane, etc)

5.- Locate the single executable file in the AutoOrtho folder and create a shortcut on your desktop.

6.- Create a folder with a name similar to AutoOrthoCache on a disk with good space and access speed. All the necessary image cache will be loaded there. This is very important. Access capacity and speed.

7.- Run, from the desktop shortcut, the AO. This process will take you, first of all, to the configuration screen that has two tabs. In the first tab [SETUP] you must enter the following data:

to. The folder where your scenarios folder is located.

Usually in:
C:\X-Plane 12\Custom Scenery or
C:\Program Files (x86)\Steam\steamapps\common\X-Plane 12
b. The folder where the cache images that AO needs and mentioned in point 6 will be stored.

c. The temporary folder for the installation (I think it brings it by default). It is usually found at:

C:\Users\<Username>\.autoortho-data\downloads

d. The Source of the images you are going to use (it is suggested to use BI which is BING) but you can try something else.

and. The maximum capacity that the image cache folder mentioned in point 6 will contain (it is suggested not less than 100 Mb). This depends on your disk space. The more images you can cache, the better your system will perform.

DON'T FORGET TO PRESS THE [SAVE] BUTTON before going to the next tab.

8. Go to the [SCENERY] tab and select the area to host AutoOrtho. It is suggested to start with the continent where you normally fly. This is done using the [INSTALL] button.

The installation of the AutoOrtho scenarios takes a long time and depends on the speed of Internet access. Be patient and wait for it to finish.
For example, in the case of South America, there are more than 8,700 Mb of files.

Once the installation is finished you can press the [RUN] button.

9.- At this point, a minimum of 3 folders will be generated in Custom Scenery. These are the following:

yAutoOrtho_Overlays
z_autoortho

and a third virtual folder that will exist only while AO is running.

This folder can have a name like “z_ao_sa” and I guess if you use another contain it will be called “z_ao_na” (North America), “z_ao_au” (Australia), etc.
The truth is that I do not know since I have only one scenario installed.

10.- This process ends when you see the message that all the necessary files have finished downloading on the command execution screen (this screen is present while AO is running).

11.- Launch X-Plane and wait for it to finish loading, with the plane and airport of your choice. It does not matter which one. Make sure your airport is within the chosen continent. Don't be surprised if you notice that the airport is gone or missing things. It is not a problem.
With the simulator open, find and copy to another folder the scenery_pack.ini file found in the Custom Scenery folder (we suggest in the DOWNLOADS or DOCUMENTS folder). Close the simulator, without closing AO.

12.- Edit the scenery_pack.ini file, from where I copied it in the previous point (this must be done with notepad). Locate the lines:

SCENERY_PACK Custom Scenery/yAutoOrtho_Overlays/
SCENERY_PACK Custom Scenery/z_ao_sa/
SCENERY_PACK Custom Scenery/z_autoortho/

Put them, in the same order as above but at the end of the scenery_pack.ini file and save the changes.

Standing over the scenery_pack.ini file, use the right mouse button to bring up the context menu, choose the file PROPERTIES option, and check the read-only box.

The idea is that this file, from now on, cannot be accessible for changes, in any way, by X-Plane. Only you will be able to alter it to add some scenario.
THIS IS VERY IMPORTANT.

Remember the order in which the scenes go in the scenery_pack.ini:

I
1000Version
SCENERY

Here are apps like SAM:

SCENERY_PACK C:\Program Files (x86)\Steam\steamapps\common\X-Plane 12\Resources\plugins\SAM\lib\SAM_GlobalTrees/
SCENERY_PACK Custom Scenery/SAM_DeveloperPack/

From here all the folders that contain scenarios. For example:

SCENERY_PACK Custom Scenery/Aerosoft - EDLP Paderborn-Lippstadt/
SCENERY_PACK Custom Scenery/Aerosoft - LFMN Nice Cote d Azur X/

From here and at the end of the scenarios the landmarks:

SCENERY_PACK Custom Scenery/X-Plane Landmarks - Berlin and Frankfurt/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Budapest/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Chicago/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Dubai/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Las Vegas/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - London/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Los Angeles/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - New York/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Paris/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Portland/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Rio De Janeiro/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Saint Louis/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - San Francisco/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Sydney/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Washington DC/

Then the simulator's own scenarios (example from X-plane 12:

SCENERY_PACK *GLOBAL_AIRPORTS*

(in X-Plane 11 it appears differently)

Then, all the necessary libraries for the custom scenarios to work, such as:

SCENERY_PACK Custom Scenery/ NAPS_library/
SCENERY_PACK Custom Scenery/ OpenSceneryX/

And finally the entries for AO should go:

SCENERY_PACK Custom Scenery/yAutoOrtho_Overlays/
SCENERY_PACK Custom Scenery/z_ao_sa/
SCENERY_PACK Custom Scenery/z_autoortho/

13.- Copy the scenery_pack.ini file into the [CUSTOM SCENER] folder of the simulator. Windows will ask if you want the old file to be overwritten with the new one. THIS IS IMPORTANT. Verify that scenery_pack.ini is READ-ONLY.

14.- Start the simulator again. A little window will appear with a warning, informing you that you cannot access the scenery_pack.ini file (you already know that because we prepared it that way).

If this warning does not appear, start again from point 11 of this instruction.

If, on the contrary, it appears, it means that you have reached this point well.

Now it's going to take a bit longer to open the simulator, since AO is downloading the images and doing its job. Do not despair. Have patience.

If your machine has good processing power and fast drives, you will be able to enjoy X-Plane and AO.

Final tip: Before flying from one point to another, first position your aircraft at point A. Then do the same for point B. Once these processes are complete, fly from A to B.
It took me a whole morning to “update” all my favorite airports with AO. I hope it will be you useful.

(sorry for my english... I use Google tralslator)

Regards
carlos
